components/Templates/Templates.php

```diff
@@ -395,6 +395,23 @@ public function add_meta_boxes() {
 					'restrict_capability' => true,
 				),
 			),
+			array(
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 4bcd1674b33f5e353e0724ee4d2728ed</sub>